test/cmdlineTests.sh

```diff
@@ -240,9 +240,9 @@ EOF
         sed -i.bak -e 's/\(\\"version\\":[ ]*\\"\)[^"\\]*\(\\"\)/\1<VERSION REMOVED>\2/' "$stdout_path"
         rm "$stdout_path.bak"
     else
-        sed -i.bak -e '/^Warning: This is a pre-release compiler version, please do not use it in production./d' "$stderr_path"
+        sed -i.bak -e '/^Warning: This is a pre-release compiler version, please do not use it in production./,+1d' "$stderr_path"
```

Comment:
What does this do :-)?

Reply:
And why is that something we *want* to do?

Reply:
It fixes the problem you pointed out above: https://github.com/ethereum/solidity/pull/14591#discussion_r1350392159

Errors and warnings gathered during compilation are always printed separated with a single empty line. If we remove the warning, but not the empty line, that line will be there for pre-release versions but not for release versions. This makes the stripped CLI output inconsistent between `force_release` jobs and other jobs.

Reply:
Ah, yeah, sorry, I didn't re-read that after being resolved :-).